src/vaults/ProofsFetcher.ts

```diff
@@ -0,0 +1,29 @@
+import { getAllProofs, Proofs } from "../utils/getCurrentOnChainDistribution";
+import { EpochConfig } from "../ages";
+import { getEpochFromId } from "../utils/timestampToEpoch";
+
+export interface ProofsFetcherInterface {
+  fetchProofs: (address: string, epochToId?: string) => Promise<Proofs[]>;
```

Comment:
```suggestion
  fetchProofs: (address: string, lastEpochId?: string) => Promise<Proofs[]>;
```

The naming is a bit confusing, especially since you have `getEpochFromId` in the same file. I was expecting something about a conversion between epoch number and epoch Id

Reply:
mmh to me lastEpoch refers to the global last epoch, where epochTo is one finished epoch but not necessary the last one (example: Any user want to verify morpho distribution for `age1-epoch3`)